.yamllint

```diff
@@ -11,3 +12,5 @@ rules:
   # NOTE(retr0h): Templates no longer fail this lint rule.
   #               Uncomment if running old Molecule templates.
   # truthy: disable
+ignore: |
```

Comment:
Is there a reason why we'd want to ignore the YAML files in this directory?

Reply:
Github uses some syntax that yamllint thinks is problematic. For example [on:](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#on) is thought as a boolean by yamllint

Reply:
The `on` issue is presented as a warning, and not an error, right?

Reply:
```
$ yamllint .github/workflows/ci.yml 
.github/workflows/ci.yml
  3:1       warning  truthy value should be one of [false, true]  (truthy)

$ echo $?
0
```

Reply:
For the rules you think don't apply, you can exclude them in this file (rather than the entire .github directory).

```
  truthy:
    check-keys: false
```

Reply:
> The `on` issue is presented as a warning, and not an error, right?

Without ignoring the .github folder, `ansible lint` exit code is `2`

Reply:
Cool. I'm OK with this new approach: ignoring just the truthy tests, and not all the tests.